src/components/Menu.scss

```diff
@@ -0,0 +1,73 @@
+.Menu{
+  position: absolute;
+  width: 398px;
+  height: 896px;
+  left: 16px;
+  top: 0px;
+  
+  /* White */
+  background: #FFFFFF;
+  box-shadow: -4px 2px 8px rgba(0, 46, 93, 0.08);
+}
+
+.Title {
+  width: 97px;
+  height: 28px;
+
+  font-family: 'Arial';
```

Comment:
I believe the font family might not be Arial. If you don't have access to the font, ask the tribute managers for it 😊